### DISCUSSION TITLE
INITIAL PROPOSAL: CRO Community Burn Cronos POS Chain

### DISCUSSION THREAD

**Discussion by zkaizhi** [crypto-org-chain/chain-main#1018] [Ideas]
### Background 
As many of you will recall, back in May, the Cronos PoS chain (formerly known as Crypto.org) passed a pivotal [governance proposal](https://www.mintscan.io/crypto-org/proposals/16) aimed at slowing down CRO token inflation. This proposal introduced a burning mechanism to reduce the overall inflation rate. The approach involves allocating a share of tokens generated through future inflation to a "Community Pool," which are then periodically burned.

### The Burn Procedure Explained
The focus of this proposal is to burn the current community pool balance and to discuss future methods of burning.
This is step 6 in the token economic cycle diagram, which we've [previously](https://medium.com/@crypto.org/a-straightforward-proposal-to-slow-down-cro-inflation-and-bring-it-closer-to-historical-levels-4355278f2de3) discussed. The idea here is to allocate a portion of tokens generated through emissions to a community pool, followed by a periodic burn. To perform this "burn," a "community pool spend" governance proposal must be initiated. Funds can then be transferred to a "dead" address which no one controls (i.e. `cro1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqtcgxmv`) , effectively reducing the available token supply and concluding a single burning cycle.

As of the time of writing, the community pool holds approximately [48 million of CRO](https://www.mintscan.io/crypto-org/address/community-pool). We project this figure to swell to around 50 million by begin-December, which brings us to our next topic.

### When do we expect to burn?
We propose to initiate **the first-ever CRO Burn when the community pool hits the 50M CRO threshold**. Achieving this target will not only be a pivotal moment in the upgrade of our protocol but will also serve as a harbinger for a more balanced and resilient token economic model for Cronos PoS. With the current rate of CRO accumulation in the community pool, we expect to reach the 50M threshold around begin-December this year.

### Future burning discussions
This inaugural CRO Burn is more than a one-off event; it's the first step in a broader, long-term strategy. As we plan for the future, we are exploring two potential paths to sustainably implement future CRO burns:

- Threshold-Triggered Burn: With this model, each time the community pool's CRO balance reaches a specified amount— e.g. 25M CRO—a burn event will be triggered. This approach ensures that burning is responsive to the rate of token accumulation, thus directly impacting the circulating supply in a dynamic manner.|

- Calendar-Governed Burn: Alternatively, we could stick to a time-based approach where burns occur at quarterly intervals, irrespective of how much CRO has been amassed in the community pool. This would provide a predictable and consistent burning schedule, which may be easier for token holders and market analysts to anticipate.

Each option has its merits, and the choice between them will set the course for how the Cronos PoS chain manages its token supply moving forward. We are eagerly looking forward to the community's insights and active participation in making this decision. Please [visit the Cronos Discord server to share your thoughts](https://discord.com/invite/cronos).

### Let us know your thoughts
We cannot stress enough the importance of active community participation in shaping the future of Cronos PoS chain. Engage with your delegated validators and take part in the proposal discussions on Discord and GitHub. Your voice is crucial in evolving a vibrant and enduring token economy that benefits all CRO holders and the broader ecosystem.

Previous blog: [A Straightforward Proposal to Slow Down CRO Inflation](https://medium.com/@crypto.org/a-straightforward-proposal-to-slow-down-cro-inflation-and-bring-it-closer-to-historical-levels-4355278f2de3)

**Links**
Initial Burn Proposal
https://github.com/crypto-org-chain/chain-main/discussions/941 
Mint Scan official proposal
https://www.mintscan.io/crypto-org/proposals/16 
Community Pool
https://www.mintscan.io/crypto-org/address/community-pool

**Comment by pmbrsantos** [comment]
If I were to choose and going forward, I would prefer to go the dynamic way, and not having a fixed calendar for the burns:
Threshold-Triggered Burn: With this model, each time the community pool's CRO balance reaches a specified amount— e.g. 25M CRO—a burn event will be triggered. This approach ensures that burning is responsive to the rate of token accumulation, thus directly impacting the circulating supply in a dynamic manner.

This would also decentralize the burn and make it automatic, so in the eyes of any external entity (like the SEC), the burn isn't controlled by anyone, rather an intrisic characteristic of the network.

**Reply by zkaizhi** [reply]
Hi @pmbrsantos ,

Thanks a lot for this feedback, we indeed see interest in both approaches, but you raise a very good point to the fact that it is automatic based on token accumulation and completely uncontrolled by anyone, making this a favourable and more decentralized way of executing the burn! We will take this seriously into consideration amongst the other feedback. 

Thanks again!

We do want to note that in principle anyone can raise a proposal on how to utilize the community tax that benefits the ecosystem!

**Comment by Markmanus** [comment]
I would prefer Treshold based Burn as it feels a bit more dynamic and also more decentralized. It responds better to market conditions and ensures that we always burn enough token to handle inflation.

Calendar based does not feel for me the right choice as our market is very volatile and unpredictable.

Thank you.

**Reply by MindlessSteel** [reply]
I vote for the threshold version being a CRO holder and family member myself, I have just became aware of this proposal today I didn't know that CRO's were being sent to burn at 25 or 50 million. After reading this proposal I definitely am form it don't think a calendar automatic burn day is the way to go in my opinion.
